content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -25,38 +26,39 @@ finally being signed.
 {{% capture body %}}
 ## Request signing process
 
-The _CertificateSigningRequest_ resource type allows a client to submit an x509 CSR (`spec.request`)
-and request it be signed by a specific _signer_, denoted by the `spec.signerName` field.
+The _CertificateSigningRequest_ resource type allows a client to ask for an X.509 certificate
+be issued, based on a signing request.
+The CertificateSigningRequest object includes a PEM-encoded PKCS#10 signing request in
+the `spec.request` field. The CertificateSigningRequest denotes the _signer_ (the
+recipient that the request is being made to) using the `spec.signerName` field.
 
 Once created, a CertificateSigningRequest must be approved before it can be signed.
-Depending on the signer requested, a CertificateSigningRequest may be auto-approved by a
-{{< glossary_tooltip text="controller" term_id="controller" >}}.
+Depending on the signer selected, a CertificateSigningRequest may be automatically approved
+by a {{< glossary_tooltip text="controller" term_id="controller" >}}.
 Otherwise, a CertificateSigningRequest must be manually approved either via the REST API (or client-go)
-or via `kubectl certificate approve`. Likewise, a CertificateSigningRequest may also be denied, in which
-case the configured signer must not sign the request.
+or by running `kubectl certificate approve`. Likewise, a CertificateSigningRequest may also be denied,
+which tells the configured signed that it must not sign the request.
 
-Once approved, a signing controller must attempt to sign the request. A signer should apply its own policy
-restrictions, similar to the approval process, on the request to ensure it never signs an invalid request.
-There is currently not a mechanism for a signer implementation to report its inability to sign a request.
-Once the signer has determined that a request should be signed, it must sign the request and store the
-resulting certificate (and if appropriate, certificate chain) in the `status.certificate` field as PEM-encoded
-certificates.
+For certificates that have been approved, the next step is signing. The relevant signing controller
+first validates that the signing conditions are met and then creates a certificate.
+The signing controller then updates the CertificateSigningRequest, storing the new certificate into
+the the `status.certificate` field of existing CertificateSigningReques object. The
+`status.certificate` field is either empty or contains a X.509 certificatem, encoded in PEM format.
```

Comment:
I don't understand the statement about it being empty in the section about the signer populating it.

Reply:
I'm trying to convey that you can have a Pending / Denied CSR and if you do then there won't be a certificate.

Reply:
Ah, maybe state that separately then?

```
The signing controller then updates the CertificateSigningRequest, storing the new certificate into
the `status.certificate` field of the existing CertificateSigningRequest object, encoded in PEM format.
The CertificateSigningRequest `status.certificate` field is empty until the signer does this.
```

Reply:
```suggestion
`status.certificate` field is either empty or contains a X.509 certificate, encoded in PEM format.
```

Reply:
Think this is sorted.